apps/web/src/pages/dao/[network]/[token]/[tokenId].tsx

```diff
@@ -124,16 +123,13 @@ const TokenPage: NextPageWithLayout<TokenPageProps> = ({
         slug={url}
         description={ogDescription}
       />
-      {token && addresses?.auction ? (
-        <Auction
-          chain={chain}
-          auctionAddress={addresses.auction}
-          collection={collection}
-          token={token}
-        />
-      ) : (
-        <AuctionSkeleton />
-      )}
+
+      <TopSection
```

Comment:
Can this have a slightly more descriptive name. Maybe just DaoTopSection or something since its specific to the dao page